containers/latest/register.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 QEMU_BIN_DIR=${QEMU_BIN_DIR:-/usr/bin}
-
+QEMU_SUFFIX=${QEMU_SUFFIX:--static}
```

Comment:
qemu-user-static shoud be statically linked. Simply rename qemu-user to qemu-user-static will not work as expected. So -static suffix is not optional.

Reply:
> `qemu-user-static` shoud be statically linked. Simply rename `qemu-user` to `qemu-user-static` will not work as expected. So `-static` suffix is not optional.

This was the default behavior before. I just added the variable default value so that it can be configured through variables.

Reply:
I don't think the default value should be changed. `-static` is necessary for `binfmt` to work properly.